calico_node/startup/startup_test.go

```diff
@@ -128,6 +136,13 @@ var _ = Describe("FV tests against a real etcd", func() {
 						Expect(pool.Spec.IPIP.Enabled).To(BeTrue())
 						Expect(pool.Spec.IPIP.Mode).To(Equal(ipip.Mode(expectIpv4IpipMode)))
 					}
+
+					if expectedIPV4NATOutgoing == false {
```

Comment:
I think it would be better to replace this if statement and the nested checks with
`Expect(pool.Spec.NATOutgoing).To(Equal(expectedIPv6NATOutgoing))`